.github/workflows/pages.yaml

```diff
@@ -1,13 +1,11 @@
 name: Release docs
 
 on:
-  workflow_dispatch:
   push:
     branches:
-      - master
-      - 'branch-**'
```

Comment:
What if I want to update documentation only one of previous versions? This won't trigger page update.

Reply:
When the GitHub Action is triggered, the ``make multiversion`` command builds docs for all versions listed in [conf.py](https://github.com/scylladb/scylla-operator/pull/324/files/217cbdf38e0932c06c2518c883e184abd83e1337#diff-008dcb3426febd767787b1521f1fe33086313b927ea37eaab86df5fa88a51698R155). For that reason, there is no need to trigger builds per branch.

This file was wrongly copied from the project `scylla-manager`, but these scripts were customized and present [some downsides](https://github.com/scylladb/scylla-manager/issues/2436). Instead, I should have copied them from the `sphinx-scylladb-theme` project to follow how are we [building docs](https://github.com/scylladb/sphinx-scylladb-theme/blob/master/.github/workflows/pages.yml) for every other project.